src/manipulation.js

```diff
@@ -159,6 +159,10 @@ function fixInput( src, dest ) {
 }
 
 jQuery.extend({
+	htmlPrefilter: function( html ) {
```

Comment:
I think this is a case of YAGNI. The jQuery analogy that comes to mind for me is the special events interface. There just aren't a lot of clients who need to use it, so coming up with a management interface for them is a lot of pain for no gain.

If you are writing a web page that incorporates lots of different code from relatively untrusted third-party sources, you will want to keep your one-and-only `htmlPrefilter` in place for the lifetime of the page because you don't trust that code to do the right thing and want to watch over it. In that case, having some other app mess with the `htmlPrefilter` is not good at all because you probably don't trust that code either.

If you are writing some kind of SPA then either the framework you are using should be ensuring sanitized HTML (and might provide hooks for that) or you will again be just doing one `htmlPrefilter` and keeping it installed permanently. I don't see the need for multiplexed functions here either.